modular_skyrat/modules/stasisrework/code/medical_designs.dm

```diff
@@ -9,9 +9,9 @@
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
 
 /datum/design/board/stasissleeper
```

Comment:
Thought I might leave them (the boards are also available now) in case folks wanted to use them for aesthetics or for a stopgap before upgrading on the maps that still have them. Would you rather they just be ripped out?

Reply:
I personally never really liked them, due to the controls being really awkward and counter-intuitive, and because of how easy it is to forget someone in there.

I wouldn't be against us gutting those sleepers away, tbh. I'll talk with the other maintainers to see what we think about it.